chart/localdb/charts/postgresql/values.yaml

```diff
@@ -15,7 +15,7 @@ global:
 image:
   registry: docker.io
   repository: bitnami/postgresql
-  tag: 11.15.0-debian-10-r93
```

Comment:
Shouldn't we change the value in the `values-production.yaml` as well? [Link](https://github.com/kyma-incubator/compass/blob/main/chart/localdb/charts/postgresql/values-production.yaml#L17)